tools/runner.py

```diff
@@ -0,0 +1,374 @@
+# Copyright: (c) Ansible Project
+# Copyright: (c) 2020, Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+import argparse
+import glob
+import json
+import os
+import shlex
+import shutil
+import subprocess
+import sys
+
+
+SEPARATOR = '=========================================================================='
+DEFAULT_PYTHON = '3.7'
+
+COLORS = {
+    'emph': 1,
+    'gray': 37,
+    'black': 30,
+    'white': 97,
+    'green': 32,
+    'red': 31,
+    'yellow': 33,
+}
+
+
+def colorize(text, color, use_color=True):
```

Comment:
```suggestion
def colorize(text, color, use_color=True):
```
it is in the run.py as well (and COLORS dict, btw), isn't it? can't we move them to somethere and import instead of having the same stuff in both files?

Reply:
I replicated them so I don't have to worry about Python import paths. Also, given the (hopefully) temporary lifetime of this product, I think this is acceptable as long as there isn't much more shared code :)